### ISSUE TITLE
Wrong accidental is rendered

### ISSUE THREAD

**Issue by infojunkie** [opensheetmusicdisplay/opensheetmusicdisplay#1074]
In [this MusicXML file](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/7386413/hamama-beida-rast.zip), a `<accidental>slash-flat</accidental>` is inserted, but when uploaded to the OSMD demo, it is rendered instead as a `quarter-flat` instead. You can [see the difference between them here](https://www.w3.org/2021/06/musicxml40/musicxml-reference/examples/key-element-non-traditional/).

**Comment by sschmidTU** [comment]
Yes, the problem here is that both `quarter-flat` and `slash-flat` have the same `alter` value of -0.5 here. So we can't just remember the accidental by its alter value, which we usually do. Musescore does the same mistake by the way.
